lib/linguist/generated.rb

```diff
@@ -683,6 +684,14 @@ def generated_html?
       end
     end
 
+    # Internal: Is this a generated jOOQ file?
+    #
+    # Return true or false
+    def generated_jooq?
+      return false unless extname.downcase == '.java'
+      lines.first(3).any? { |l| l.include? 'This file is generated by jOOQ.' }
```

Comment:
If the header is always the same, let's limit this to the first two lines.

Reply:
done!